file-check/action.yaml

```
@@ -0,0 +1,54 @@
+name: file-check
```

Please change the directory name and other places as well.

```suggestion
name: check-file-existence
```

---

Changed b13e007

---

Thank you!